src/System.Private.ServiceModel/tools/IISHostedWcfService/Web.config

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <!--Copyright (c) Microsoft Corporation.  All Rights Reserved.-->
+<configuration>
+  <system.diagnostics>
+    <sources>
+      <source name="System.ServiceModel" switchValue="Information, ActivityTracing" propagateActivity="true">
+        <listeners>
+          <add name="traceListener" type="System.Diagnostics.XmlWriterTraceListener" initializeData="c:\log\Traces.svclog" />
```

Comment:
From my investigation:
1. When the same file name specified, the subsequent service instances will get a trace file name prefix with a Guid automatically. I.E we can use the same file name, in the run time, a different file name will be automatically generated per service instance.
2. Relative to the root of repo is impossible to predefine in the config file, and it's also not recommended to put log file under vroot for the security reason.